lib/cli/kit/system.rb

```diff
@@ -191,9 +191,16 @@ def resolve_path(a, env)
           # If only one argument was provided, make sure it's interpreted by a shell.
           return ["true ; " + a[0]] if a.size == 1
           return a if a.first.include?('/')
-          item = env.fetch('PATH', '').split(':').detect do |f|
-            File.exist?("#{f}/#{a.first}")
+
+          # If executing "foo", and no foo executable was found on the PATH, shells will
+          # see if one exists in the current directory and execute that.
```

Comment:
I don't think this comment is true anymore.

Reply:
😅